nemo_curator/classifiers/base.py

```diff
@@ -121,44 +121,29 @@ def _run_classifier_helper(
     prob_col: str = None,
 ) -> "dask_cudf.DataFrame":
 
-    keep_prob = prob_col is not None
-    prob_internal_col = "_prob"
-    # TODO: Make crossfit handle this cleanly
-    pred_internal_col = "labels"
-    df["sliced_text"] = df[text_field].str.slice(0, max_chars)
+    if prob_col:
+        df[prob_col] = 0
```

Comment:
I wanted CrossFit to internally create the `prob_col`, but I was having trouble getting this to work. Specifically, I was able to get the entire classification pipeline to work, but then at the end `prob_col` would be dropped somewhere and not returned with the result.